bitnami/harbor/templates/notary/notary-server.yaml

```diff
@@ -51,33 +51,34 @@ spec:
             - name: DB_URL
               value: {{ include "harbor.database.notaryServer" . }}
           volumeMounts:
-            - name: notary-config
-              mountPath: /etc/notary
+            - name: config
+              mountPath: /etc/notary/server-config.postgres.json
+              subPath: server.json
             - name: root-certificate
```

Comment:
This will through an error because you changed the name to the volume.

Reply:
Sorry about this, I had notary disabled and overlooked this. It is fixed now